web/ui/templates/_base.html

```diff
@@ -40,8 +40,9 @@
             <li><a href="{{ pathPrefix }}/alerts">Alerts</a></li>
             <li><a href="{{ pathPrefix }}/graph">Graph</a></li>
             <li><a href="{{ pathPrefix }}/status">Status</a></li>
```

Comment:
The second one is 'Targets', the main status one would link to it and may include summary target information (e.g. total number). Over time everything gets split out, and we're left with just summary information.